models/ActivePartner.v1.json

```diff
@@ -16,6 +16,9 @@
     "contact": {
       "$ref": "./Contact.v1.json"
     },
+    "main_contact": {
```

Comment:
Do we need main contact in the active partner ?

Reply:
Yes, because I use it in the PDF, for legal reasons: there are sentences like "Hypo, représenté by Fabrice H. ..."